book/modules/appendix2.tex

```diff
@@ -68,8 +93,34 @@
 	\begin{example}
 		Which of the follow matrices are in reduced row echelon form? For those that are, identify which
 		columns are pivot columns. For those that are not, what condition(s) fail?
-
-		XXX Finish include examples of 4 matrices, one of which is in RREF
+		\[
+			A=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 2\\
+			0 & 0 & 0 & 0\\
+			0 & 0 & 1 & 7
+			\end{bmatrix},\,
+			B=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 8\\
+			0 & 1 & 3 & 7\\
+			0 & 2 & 1 & 4\\
+			\end{bmatrix},\,
+			C=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 2\\
+			0 & 1 & 0 & 1\\
+			0 & 0 & 1 & 8
+			\end{bmatrix},\,
+			D=
+			\begin{bmatrix}[cccc]
+			0 & 1 & 3 & 6\\
+			1 & 0 & 0 & 9\\
+			0 & 0 & 1 & 4\\
+			\end{bmatrix}
+		\]
+		
+		$C$ is in reduced row echelon form and column 1, 2, 3 are the pivot columns. The fourth property fails for $A$. The first and second property fails for $B$. The second and third property fails for $D$.
```

Comment:
Explain these in order, and in more detail (for example by highlighting in color the columns, etc that are incorrect). Keep in mind, this is the *very first* time students are seeing rref.